docs/source/model_doc/speech_to_text_2.mdx

```diff
@@ -69,7 +69,7 @@ predicted token ids.
 >>> ds = ds.map(map_to_array)
 
 >>> inputs = processor(ds["speech"][0], sampling_rate=16_000, return_tensors="pt")
->>> generated_ids = model.generate(input_ids=inputs["input_values"], attention_mask=inputs["attention_mask"])
+>>> generated_ids = model.generate(inputs=inputs["input_values"], attention_mask=inputs["attention_mask"])
```

Comment:
This is the function generate so the input parameter should be `input_ids`.
It's different from the class `transformers.SpeechEncoderDecoderModel`

Reply:
Thanks for fixing this!